### PR TITLE
♻️(frontend) update account page

### PR DESCRIPTION
**Pull request by PanchoutNathan**
## Purpose

Some features such as avatar upload or account deletion are not yet available. They are therefore not displayed to the user.

## Proposal

- [x] remove avatar picker
- [x] remove delete account block

### CODE REVIEW

File: src/frontend/magnify/src/i18n/TranslationProvider/TranslationsProvider.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Box, Spinner } from 'grommet';
```
Comment (jbpenrath) [inline src/frontend/magnify/src/i18n/TranslationProvider/TranslationsProvider.tsx:1]: You have to drop this commit

File: src/frontend/demo/src/views/profile/account/index.tsx [hunk 2]
```diff
@@ -1,9 +1,4 @@
-import {
```
Comment (jbpenrath) [inline src/frontend/demo/src/views/profile/account/index.tsx:1]: There is a 🔥 gitmoji to type commit which deletes code.

File: src/frontend/magnify/src/components/profile/IdentityBlock/IdentityBlock.tsx [hunk 3]
```diff
@@ -17,9 +16,9 @@ export default function IdentityBlock({ margin = { vertical: 'small' } }: Identi
   return (
     <Card background="white" margin={margin} pad={isSmall ? 'medium' : 'small'}>
       <Box align={'center'} direction={isSmall ? 'column' : 'row'} justify={'center'}>
-        <Box margin={{ vertical: 'auto', horizontal: 'medium' }}>
-          <AvatarForm key={user?.id || ''} id={user?.id} />
-        </Box>
+        {/*<Box margin={{ vertical: 'auto', horizontal: 'medium' }}>*/}
+        {/*  <AvatarForm key={user?.id || ''} id={user?.id} />*/}
+        {/*</Box>*/}
```
Comment (jbpenrath) [inline src/frontend/magnify/src/components/profile/IdentityBlock/IdentityBlock.tsx:21]: Hmmm... I assume this code is removed temporarily but why not remove it completely then you will be able to find this commit from this commit later, WDYT ?

If you prefer to let this code commented, that's ok but add a comment to explain why this code is currently disabled.

File: src/frontend/magnify/src/i18n/TranslationProvider/TranslationsProvider.tsx (hunk 1 above)
Comment (PanchoutNathan) [reply src/frontend/magnify/src/i18n/TranslationProvider/TranslationsProvider.tsx:1]: Resolve when merge the root branch

**Pull request merged**